ci/gen_commit_pragmas.py

```diff
@@ -88,7 +195,22 @@ def gen_commit_pragmas(target):
     if not modified_files:
         return pragmas
 
-    test_tag = ' '.join(sorted(tags.files_to_tags(modified_files)))
+    commit_pragma_mapping = read_commit_pragma_mapping()
+
+    def __pragma_config(pragma_key, default):
+        """Return the configs for a single pragma key.
```

Comment:
Do we need Args and Returns description here?

Reply:
For locally scoped helper functions I think it's less important. But I'll just add anyway

Reply:
Added Args, Returns, and a better example